src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -404,20 +386,19 @@ export const TradeInput = memo(() => {
       // don't allow users to execute trades while the quote is being updated
       isTradeQuoteApiQueryPending[activeSwapperName] ||
       // don't allow users to proceed until all swappers have an initial result
```

Comment:
Remove me? Since this referred to `isQuoteRequestIncomplete` below previously

Reply:
updated comment to make sense f3f3a014b7e4b7338311a9ef85683daaa8cb9fef